neptune/client.py

```diff
@@ -558,35 +548,36 @@ def _convert_to_api_properties(self, raw_properties):
             ) for key, value in raw_properties.items()
         ]
 
-    def _convert_experiment_to_leaderboard_entry(self, experiment):
+    def _convert_to_experiment(self, experiment):
```

Comment:
Maybe rename it to `api_experiment`? `convert_to_experiment(experiment)` may be confusing.